pkg/connector/user.go

```diff
@@ -246,10 +247,34 @@ func listUsers(ctx context.Context, client *okta.Client, token *pagination.Token
 	if qp.Search == "" {
 		qp.Search = "status pr" // ListUsers doesn't get deactivated users by default. this should fetch them all
 	}
-	oktaUsers, resp, err := client.User.ListUsers(ctx, qp)
+
+	uri := usersUrl
+	if qp != nil {
+		uri += qp.String()
+	}
+
+	reqUrl, err := url.Parse(uri)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	oktaUsers := make([]*okta.User, 0)
+	rq := client.CloneRequestExecutor()
+	req, err := rq.
+		WithAccept(ContentType).
+		WithContentType(`application/json; okta-response="omitCredentials,omitCredentialsLinks,omitTransitioningToStatus"`).
+		NewRequest(http.MethodGet, reqUrl.String(), nil)
 	if err != nil {
-		return nil, nil, handleOktaResponseError(resp, err)
+		return nil, nil, err
 	}
+
+	req.Header.Set("Content-Type", `application/json; okta-response="omitCredentials,omitCredentialsLinks,omitTransitioningToStatus"`)
```

Comment:
I had to set it here because it was still including the credentials when setting it with `WithContentType`

Reply:
It's probably a good idea to put a comment here explaining why this line is here.